Libraries/test/Amazon.Lambda.AspNetCoreServer.Test/TestApiGatewayHttpApiV2Calls.cs

```diff
@@ -235,6 +235,22 @@ public async Task TestMultipleCookie()
             Assert.Equal("TestValue3", response.Body);
         }
 
+        [Fact]
+        public async Task TestTraceIdSetFromLambdaContext()
+        {
+            Environment.SetEnvironmentVariable("_X_AMZN_TRACE_ID", "MyTraceId-1");
```

Comment:
1. shouldn't manipulate the real environment variable ideally, creating a wrapper around will solve this
2. If we do manipulate real env var, it makes sense to clear them out to be on safe side

Reply:
I agree it would be good to have environment variables be mockable in this library but it just isn't setup that way right now to pass around an environment variable abstractions through out the code. I'm not going to do a major refactor like that right now.

I did add an explicit clear of the environment variable in the test.